src/main/java/com/trailfinder/dto/EventDTO.java

```diff
@@ -137,5 +137,10 @@ public void setEventCreator(ProfileDTO eventCreator) {
 	public void setAttendees(List<ProfileDTO> attendees) {
 		this.attendees = attendees;
 	}
+	
+	@Override
+	public String toString() {
+		return "New event at " + this.latitude + " " + this.longitude;
```

Comment:
Overriding toString is a good idea... though I'd remove the word "New".  This could be invoked at any time; not just when the event is created.